src/ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -134,6 +134,20 @@ public T Value
         }
 
         /// <summary>
+        /// Gets a boolean value indicating whether the ObservableAsPropertyHelper
+        /// has subscribed to the source Observable.
+        /// Useful for scenarios where you use deferred subscription and want to know if 
+        /// the ObservableAsPropertyHelper Value has been accessed yet.
+        /// </summary>
+        public bool IsSubscribed
```

Comment:
I know we have no consistency in this regard, but should we just use C# 7 expression-bodied property here? 7 lines will reduce to 1. Per above, I would expect to see:
```C#
public bool IsSubscribed => _activated > 0;

Reply:
I think that's actually C# 6 :)

Reply:
C# 7 would be 

public bool IsSubscribed 
{
  get => _activated > 0;
}

Reply:
Yeah, I meant 6 ;) They're all blending into one at this point 🙃